docs/UserGuide.md

```diff
@@ -1,127 +1,168 @@
- 
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
- 
-> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
-> items with `...` after them can have multiple instances. Order of parameters are fixed. 
-> 
-> Persons can have any number of tags (including 0)
-
-Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
-
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
+
+## Adding a task: `add`
+Adds a task to the task list<br>
+Format: `add TITLE [p]d/DETAILS [p]s/STARTTIME [p]e/ENDTIME [p]t/tag...`
+
+> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional,
+> items with `...` after them can have multiple instances. Order of parameters are fixed.
+>
+> Put a `p` before the details / group / tags prefixes to mark it as `private`. `private` details can only
+> be seen using the `unveil` command.
+>
+> Tasks can have any number of tags (including 0)
+
+Examples:
+* `add CS1020 Tutorial d/many questions e/051016 1200  t/needhelp`
+* `add CS2103 Project pp/hard to do s/051016 1200 e/051116 1200  t/priority1`
+
+## Listing all tasks : `list`
+Shows a list of all tasks in task list.<br>
 Format: `list`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
+## Finding all tasks containing any keyword in their name: `find`
+Finds tasks whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
-and persons matching at least one keyword will be returned (i.e. `OR` search).
+> The search is case insensitive, the order of the keywords does not matter, only the name is searched,
+and tasks matching at least one keyword will be returned (i.e. `OR` search).
+
+Examples:
+* `find cs2103`<br>
+  Returns `CS2103` and `cs2103`
+* `find cs1010 cs1020 cs2103`<br>
+  Returns Any tasks having names `cs1010`, `cs1020`, or `cs2103`
+* `find <group>/<tags>/<dates>’<br>
+  Returns Any tasks having the same group/tags/dates.
+
+##Set file location: ‘locate’
+Set the storage file location.
+Format: ‘locate [NEW FILE DIRECTORY]
+
+Example:
+* `locate Desktop/`<br>
+
+##Change command name: ‘change’
+Change the command to user’s preference
+Format : change [COMMAND WORD] [USER PREFERENCE COMMAND WORD]
+
+Example:
+* `change delete d`<br>
 
-Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+##set username: ‘username’
+Set the username to indicate the application is belong to which user
+Format: username USERS NAME
```

Comment:
Having this feature might require implementing other supportive features like listing all usernames.